src/move/04_analyze_latent/__main__.py

```diff
@@ -6,14 +6,19 @@
 from move.utils.data_utils import get_data, merge_configs
 from move.utils.visualization_utils import embedding_plot_discrete, embedding_plot_float, visualize_training, plot_reconstruction_distribs, visualize_embedding, plot_categorical_importance, plot_continuous_importance
 from move.utils.analysis import get_latents, calc_categorical_reconstruction_acc, calc_continuous_reconstruction_acc, get_embedding, get_pearsonr, get_feature_importance_categorical, get_feature_importance_continuous, save_feat_results, get_feat_importance_on_weights 
+from move.utils.logger import get_logger
 
 import numpy as np
 
-@hydra.main(config_path="../conf", config_name="main")
+@hydra.main(config_path="../conf", config_name="main", version_base="1.2")
```

Comment:
if you have the version base as a string every where (`"1,2"`) this might be annoying to update in a few months or years.